### PR TITLE
[docs][DB-5232] changes to the YCQL Snapshot based backup/restore document

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves https://yugabyte.atlassian.net/browse/DB-5232

Note to self: back-port till 2.12

Build preview : https://deploy-preview-15968--infallible-bardeen-164bc9.netlify.app/preview/manage/backup-restore/snapshots-ycql/
@netlify /preview/manage/backup-restore/snapshots-ycql/

### CODE REVIEW

File: docs/content/preview/manage/backup-restore/snapshots-ycql.md [hunk 1]
```diff
@@ -154,24 +154,28 @@ You can restore a snapshot that you have [moved to external storage](#move-a-sna
...
 1. Copy the tablet snapshots. Use the tablet mappings to copy the tablet snapshot files from the external location to an appropriate location, such as `yb-data/tserver/data/rocksdb/table-<tableid>/tablet-<tabletid>.snapshots`.<br>
 
     Based on the preceding examples, you would execute the following commands:
 
     ```sh
-    cp -r snapshot/tablet-b0de9bc6a4cb46d4aaacf4a03bcaf6be.snapshots/0d4b4935-2c95-4523-95ab-9ead1e95e794 \
-          ~/yugabyte-data-restore/node-1/disk-1/yb-data/tserver/data/rocksdb/table-00004000000030008000000000004001/tablet-50046f422aa6450ca82538e919581048.snapshots/6beb9c0e-52ea-4f61-89bd-c160ec02c729
+    scp -r /mnt/d0/yb-data/tserver/data/rocksdb/table-00004000000030008000000000004003/ \
+        tablet-b0de9bc6a4cb46d4aaacf4a03bcaf6be.snapshots/6e7e85b0-13ef-4073-9ab7-224cb77f22ef/* \
+        10.9.206.43:/mnt/d0/yb-data/tserver/data/rocksdb/table-00004000000030008000000000004001/ \
```
Comment (vpatibandla-yb) [inline docs/content/preview/manage/backup-restore/snapshots-ycql.md:167]: Change the IP to target node ip

File: docs/content/preview/manage/backup-restore/snapshots-ycql.md [hunk 2]
```diff
@@ -154,24 +154,28 @@ You can restore a snapshot that you have [moved to external storage](#move-a-sna
...
+        tablet-50046f422aa6450ca82538e919581048.snapshots/a26f19a2-ae82-4aae-810e-bb03d0fefb04/
     ```
-    
+
     ```sh
-    cp -r snapshot/tablet-27ce76cade8e4894a4f7ffa154b33c3b.snapshots/0d4b4935-2c95-4523-95ab-9ead1e95e794 \
-          ~/yugabyte-data-restore/node-1/disk-1/yb-data/tserver/data/rocksdb/table-00004000000030008000000000004001/tablet-111ab9d046d449d995ee9759bf32e028.snapshots/6beb9c0e-52ea-4f61-89bd-c160ec02c729
+    scp -r /mnt/d0/yb-data/tserver/data/rocksdb/table-00004000000030008000000000004003/ \
+        tablet-27ce76cade8e4894a4f7ffa154b33c3b.snapshots/6e7e85b0-13ef-4073-9ab7-224cb77f22ef/* \
+        10.9.206.43:/mnt/d0/yb-data/tserver/data/rocksdb/table-00004000000030008000000000004001/ \
```
Comment (vpatibandla-yb) [inline docs/content/preview/manage/backup-restore/snapshots-ycql.md:174]: Same as above

**Review by vpatibandla-yb — COMMENTED** [review]
Few minor comments

**Review by vpatibandla-yb — CHANGES_REQUESTED** [review]
Also make changes for YSQL

**Review by ag26jan — COMMENTED** [review]
Please make the changes [vpatibandla-yb](https://github.com/vpatibandla-yb) pointed. Rest looks good to me as well.

**Review by lizayugabyte — APPROVED** [review]

**Review by vpatibandla-yb — APPROVED** [review]

**Pull request merged**